ome/datasource_application_certificate_test.go

```diff
@@ -0,0 +1,43 @@
+package ome
```

Comment:
```
@@ -0,0 +1,43 @@
+package ome

+import (
+	"os"
+	"testing"

+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)

+func TestDataSource_ReadCert(t *testing.T) {
+	if os.Getenv("TF_ACC") == "" {
+		t.Skip("Don't run with unit tests because it will try to create the context")
+	}

+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testGetCert + certDataOut,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckOutput("fetched", "true"),
+				),
+			},
+		},
+	})
+}

+var certDataOut = `
+output "fetched" {
+	value = alltrue([
+		length(data.ome_application_certificate.cert.valid_to) > 0,
+		length(data.ome_application_certificate.cert.valid_from) > 0,
+		length(data.ome_application_certificate.cert.issued_to.distinguished_name) > 0,
+		length(data.ome_application_certificate.cert.issued_by.distinguished_name) > 0,
+	])
+}
+`

+var testGetCert = testProvider + `
+data "ome_application_certificate" "cert" {
+}
+`

Action:
- ```
+
```
I suggest to reorder the checks in the `Resource.TestCheckOutput` to be more readable and separated by line:
```
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckOutput("fetched", "true"),
+                // for better readability
+                )
```
This is based on my understanding of best practices for implementing Terraform providers. Let me know if you'd like me to elaborate or if there's anything else I can help with!